localisation/00_zuwc_united_localisation.csv

```diff
@@ -516,27 +516,28 @@ EVTDESC_WCFEL_201_case_03;Fel energy has left unrecoverable scars in the provinc
-EVTDESC_WCFEL_9973_deceitful_observer;You are ready for the next ritual. By sacrificing someone's life, you managed to create a tear between Azeroth and the Twisting Nether. Reality is briefly distorted, and you saw the figure of a giant levitating creature with one glaring eye. A few moments later, it materialized near you and its eye became wreathed in green flame. "I do not need you anymore, mortal!";;Ihr seid bereit f�r das n�chste Ritual. Durch das erbrachte Opfer ist es Euch gelungen, einen Riss zwischen Azeroth und dem wirbelnden Nether zu schaffen. Die Realit�t wird f�r einen kurzen Augenblick verzerrt und Ihr seht die Gestalt einer gewaltigen schwebenden Kreatur mit einem stechenden Auge. Einen Augenblick sp�ter materialisiert sie neben Euch und ihr Auge ist umgeben von gr�nen Flammen. �Ich brauche dich nicht mehr, erb�rmliche[From.GetrMale] Sterbliche[From.GetrMale]!�;;;;;;;;;;;x
-EVTDESC_WCFEL_9973_normal_case;You are ready for the next ritual. By sacrificing someone's life, you managed to create a tear between Azeroth and the Twisting Nether. Reality was briefly distorted, and you saw the figure of a large demon. It straightened up and offered you a gift in gratitude for your help before disappearing.;;Ihr seid bereit f�r das n�chste Ritual. Durch das erbrachte Opfer ist es Euch gelungen, einen Riss zwischen Azeroth und dem wirbelnden Nether zu schaffen. Die Realit�t wird f�r einen kurzen Augenblick verzerrt und Ihr seht die Gestalt eines gewaltigen D�mons. Er strafft sich und bietet Euch ein Geschenk als Dank f�r Eure Hilfe, bevor er verschwindet.;;;;;;;;;;;x
-EVTDESC_WCFEL_9973_good_case;You are ready for the next ritual. By sacrificing someone's life, you managed to create a tear between Azeroth and the Twisting Nether. Reality was briefly distorted, and you saw the figure of a large demon. It straightened up and offered you the previously agreed upon gift as well as its loyalty in gratitude for your help.;;Ihr seid bereit f�r das n�chste Ritual. Durch das erbrachte Opfer ist es Euch gelungen, einen Riss zwischen Azeroth und dem wirbelnden Nether zu schaffen. Die Realit�t wird f�r einen kurzen Augenblick verzerrt und Ihr seht die Gestalt eines gro�en D�mons. Er richtet sich auf und bietet Euch das vorher vereinbarte Geschenk und seine Unterst�tzung als Dank f�r Eure Hilfe an.;;;;;;;;;;;x
+EVTDESC_WCFEL_9973;You have begun the next ritual. By sacrificing the life of a sentient, you manage to tear open a temporary rift that bridges the gap between Azeroth and the Twisting Nether. Seconds later, you see the figure of [Root.GetDemonTypeOutcome];;Ihr habt das n�chste Ritual begonnen. Durch die Opferung eines f�hlenden Wesens gelingt es Euch, einen vor�bergehenden Riss zwischen Azeroth und dem wirbelnden Nether zu �ffnen. Sekunden sp�ter erscheint die Gestalt [Root.GetDemonTypeOutcome].;;;;;;;;;;;x
```

Comment:
Honestly, looking at it again, 'By sacrificing the life of a sentient' just sounds off. Not the most important of fixes, but I think this is better.
```suggestion
EVTDESC_WCFEL_9973;You have begun the next ritual. By sacrificing a sentient being's life, you manage to tear open a temporary rift that bridges the gap between Azeroth and the Twisting Nether. Seconds later, you see the figure of [Root.GetDemonTypeOutcome];;Ihr habt das n�chste Ritual begonnen. Durch die Opferung eines f�hlenden Wesens gelingt es Euch, einen vor�bergehenden Riss zwischen Azeroth und dem wirbelnden Nether zu �ffnen. Sekunden sp�ter erscheint die Gestalt [Root.GetDemonTypeOutcome].;;;;;;;;;;;x
```

Reply:
done.